docs/APP-CONNECTIVITY.md

```diff
@@ -229,62 +231,106 @@ are still supported and must be configured to allow anything beyond just DNS and
 Since there is no NAT between applications and external endpoints, properly configured
 inbound ACL rules are that much more important.
 
-Metadata HTTP server is run for switch network instance only if it has uplink port with
-an IP address.
+A metadata HTTP server is run for a switch network instance only if it has a port attached
```

Comment:
Was this correct before your change? I didn't think we need a (external/uplink) port, but that we do need an bridge IP address assigned to EVE, meaning that it needs to be a local NI.
If it needs to be fixed a separate PR (or commit) would make sense.

Reply:
For local network instance we indeed do not need port to start metadata server.
But for switch NI (which is described in this doc section), we start it only if we get IP address on the bridge. This happens only if NI has a (single) port attached and that port receives an IP address (statically or via DHCP, i.e. it is not set with "DHCP passthrough"). Then we install ACL to prevent access to that metadata server from outside.